### DISCUSSION TITLE
GeoJson features disappear with negative z coordinates and pitch = 0

### DISCUSSION THREAD

**Discussion by devolod** [visgl/deck.gl#8640] [Q&A] [answered]
Hi All!

I use `GeoJsonLayer` to display lines and points in `3D` view. Unfortunately, features with negative `Z` coordinates get disappeared when `pitch = 0` . For example, red dashed line on pictures below.

With pitch = 24:
![image](https://github.com/visgl/deck.gl/assets/11441427/85df16b3-fb2b-40e8-a593-a859775300df)

With pitch = 0:
![image](https://github.com/visgl/deck.gl/assets/11441427/d7fa6fab-4624-49e0-8036-89479c15567e)

Is there way to prevent this behavior?

**Comment by Pessimistress** [comment, marked as answer]
https://github.com/visgl/deck.gl/issues/4931

**Reply by devolod** [reply]
Thanks!
